index.html

```diff
@@ -65,40 +65,8 @@ <h2 class="title" id="try">
 
         <div class="has-text-centered">
             <div class="is-pulled-left download">
-                <div class="smartphone"><img class="content" aria-hidden="true" src="assets/screenshots/tower-builder.png"></div>
-                <div><a href="https://github.com/webxdc/tower-builder/releases/latest/download/tower-builder.xdc">download<br><code>tower.xdc</code></a></div>
-            </div>
-            <div class="is-pulled-left download">
-                <div class="smartphone"><img class="content" aria-hidden="true" src="assets/screenshots/hextris.png"></div>
-                <div><a href="https://github.com/webxdc/hextris/releases/latest/download/hextris.xdc">download<br><code>hextris.xdc</code></a></div>
-            </div>
-            <div class="is-pulled-left download">
-                <div class="smartphone"><img class="content" aria-hidden="true" src="assets/screenshots/chess.png"></div>
-                <div><a href="https://github.com/webxdc/ChessBoard/releases/latest/download/ChessBoard.xdc">download<br><code>chess.xdc</code></a></div>
```

Comment:
IMO Chess is a better example of a webxdc game compared to e.g. 2048.

Reply:
yes, but it's tricky to actually use chessboard because of the lack of notifications when the opponent has moved.  also it doesn't count time etc.  the other games are actually played a lot so let's put them up front.